openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -1096,13 +1094,14 @@ def run(self, *, dry=False, verbose=True,
             return {'debug': {'sampler': sampler}}
 
     @staticmethod
-    def analyse(where) -> dict:
+    def structural_analysis(scratch, shared) -> dict:
         # don't put energy analysis in here, it uses the open file reporter
         # whereas structural stuff requires that the file handle is closed
-        analysis_out = where / 'structural_analysis.json'
+        # TODO: we should just make openfe_analysis write an npz instead!
+        analysis_out = scratch / 'structural_analysis.json'
 
         ret = subprocess.run(['openfe_analysis', 'RFE_analysis',
-                              str(where), str(analysis_out)],
+                              str(shared), str(analysis_out)],
```

Comment:
Could break these list items into one on each line, put a comment next to each as to what the arguments are.

Reply:
Done, thanks!